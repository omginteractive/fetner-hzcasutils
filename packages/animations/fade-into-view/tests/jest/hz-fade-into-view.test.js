/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import FadeIntoView from '../../src';

describe('FadeIntoView', () => {
  it('should render simple text', () => {
    const item = (
      <FadeIntoView direction="Up" render={() => <p>Boo!</p>} activate />
    );
    const renderedItem = renderer.create(item).toJSON();
    expect(renderedItem).toMatchSnapshot();
  });

  it('should render link text', () => {
    const item = (
      <FadeIntoView
        direction="Up"
        activate
        render={() => <a href="http://www.google.com">Boo!</a>}
      />
    );
    const renderedItem = renderer.create(item).toJSON();
    expect(renderedItem).toMatchSnapshot();
  });
});
