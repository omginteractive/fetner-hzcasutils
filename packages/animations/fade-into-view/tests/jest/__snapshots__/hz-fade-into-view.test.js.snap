// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FadeIntoView should render link text 1`] = `
<div
  style={
    Object {
      "opacity": 0,
      "transform": "translateY(100px)",
    }
  }
>
  <a
    href="http://www.google.com"
  >
    Boo!
  </a>
</div>
`;

exports[`FadeIntoView should render simple text 1`] = `
<div
  style={
    Object {
      "opacity": 0,
      "transform": "translateY(100px)",
    }
  }
>
  <p>
    Boo!
  </p>
</div>
`;
